roles/grafana_cloud_operator/tasks/grafana_oncall_hub_spoke.yml

```diff
@@ -36,6 +36,28 @@
 
 - name: Integration creation
   block:
+    - name: Fetch Slack Channel from clusterdeployment namespace
+      kubernetes.core.k8s_info:
+        api_version: slack.stakater.com/v1alpha1
+        kind: Channel
+        namespace: "{{ item.namespace }}"
+      register: slack_channel_info
+      loop: "{{ create_integration_for }}"
+      loop_control:
+        label: "{{ item.namespace }}"
+
+    - name: Extract Channel id from slack channel
+      ansible.builtin.set_fact:
+        slack_channel_ids: "{{ slack_channel_ids | default([]) + [{'name': item.metadata.name, 'slack_id': item.status.id}] }}"
+      loop: "{{ slack_channel_info.results | map(attribute='resources') | flatten }}"
+      loop_control:
+        label: "{{ item.metadata.name }}"
+
+    - name: Print the slack id and name for each channel
```

Comment:
I suppose this is used only for debug. Remove this step if its not required.

Reply:
removed